mimecast/assets/logs/mimecast.yaml

```diff
@@ -0,0 +1,265 @@
+    source: log
+  - groups:
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Mimecast Email Security
+  enabled: true
+  filter:
+    query: "source:mimecast"
+  processors:
+    - type: pipeline
+      name: Audit
+      enabled: true
+      filter:
+        query: "service:audit"
+      processors:
+        - type: date-remapper
+          name: Define `eventTime` as the official date of the log
+          enabled: true
+          sources:
+            - eventTime
+        - type: grok-parser
+          name: Parsing Audit Event Info Attribute
+          enabled: true
+          source: eventInfo
+          samples:
+            - "White URL Created: URL: https://apple.com, Match Type: domain,
+              Action: block, Comment: demo comment, Disable Rewriting: false,
+              Disable Log Click: false, Disable User Awareness: true, Date:
+              2024-05-27, Time: 09:59:45+0000, IP: 148.139.125.20, Application:
+              CDS Demo"
+            - "Threat intel multiple feeds download  -
+              malware_grid_csv_20240527060041278.zip, Date: 2024-05-27, Time:
+              10:00:41+0000, IP: 47.198.5.185, Application: Splunk v5"
+          grok:
+            supportRules: ""
+            matchRules: 'rule %{data::keyvalue(": "," ",", ")}'
+        - type: attribute-remapper
+          name: Map `user` to `usr.name`
+          enabled: true
+          sources:
+            - user
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `IP` to `network.client.ip`
+          enabled: true
+          sources:
+            - IP
+          sourceType: attribute
+          target: network.client.ip
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Rejection
+      enabled: true
+      filter:
+        query: "service:rejection"
+      processors:
+        - type: date-remapper
+          name: Define `created` as the official date of the log
+          enabled: true
+          sources:
+            - created
+        - type: attribute-remapper
+          name: Map `ipAddress` to `network.client.ip`
+          enabled: true
+          sources:
+            - ipAddress
+          sourceType: attribute
+          target: network.client.ip
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `toAddress` to `usr.email`
+          enabled: true
+          sources:
+            - toAddress
+          sourceType: attribute
+          target: usr.email
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+    - type: pipeline
+      name: " TTP Impersonation"
```

Comment:
There is a a front space you can remove here

Reply:
Removed Space. Thanks..!!